tests/test_packages/test_connections/test_p2p_libp2p/test_communication.py

```diff
@@ -123,11 +123,10 @@ def setup_class(cls):
         cls.multiplexer1 = Multiplexer([cls.connection1])
         cls.multiplexer1.connect()
```

Comment:
Mentioned last week, but we didn't have time then: the tests should be running in a test dir (see `setup_class` where this is commented out).

Reply:
right, I can look into it as well